os-server/src/main/java/app/openschool/coursemanagement/CategoryRepository.java

```diff
@@ -1,15 +1,22 @@
 package app.openschool.coursemanagement;
 
 import app.openschool.coursemanagement.entities.Category;
+import java.util.List;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
 @Repository
-public interface CategoryRepository extends JpaRepository<Category, Long> {
+public interface CategoryRepository extends JpaRepository<Category, Integer> {
```

Comment:
Why this is changed to Integer ?

Reply:
The id type of category in DB is INT. I thought probably it is redundant to use Long type in app model.